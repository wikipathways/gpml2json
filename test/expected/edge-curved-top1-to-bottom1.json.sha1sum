f5fb82069f47d91018328bb17504cb60103b2a17 ?-
