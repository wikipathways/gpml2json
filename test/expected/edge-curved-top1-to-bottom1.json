{"pathway":{"@context":["https://cdn.rawgit.com/wikipathways/WpVocabularies/7a46a05/contexts/pvjs.jsonld"],"backgroundColor":"white","borderWidth":0,"color":"black","contains":["f8809","ad716","ea3a3"],"drawAs":"rect","gpmlElementName":"Pathway","height":825.09187756604,"kaavioType":"Group","name":"test case","padding":5,"type":["Pathway"],"width":1047.7865523249097,"x":0,"y":0,"zIndex":0,"fontSize":12,"fontWeight":"bold","textAlign":"left","verticalAlign":"top","dataSourceVersion":"20130813","standardName":"test case","displayName":"test case","textContent":"Name: test case","isSimilarTo":"http://wikipathways.org/index.php/Special:SearchPathways?query=test%20case&doSearch=1","id":"hash://sha256/7a7c237baa34145edf6aefe658151d9be2c3423cf0d5c0c024f39947575cbb32"},"entityMap":{"ad716":{"id":"ad716","textContent":"GeneProduct","wpType":"GeneProduct","x":204.46501540477257,"y":110.62567176545976,"fontSize":10,"height":20,"verticalAlign":"middle","width":80,"zIndex":32948,"textAlign":"center","color":"#000000","backgroundColor":"#ffffff","textDecoration":"none","fontFamily":"Arial","fontStyle":"normal","fontWeight":"normal","borderWidth":1,"drawAs":"rect","lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"type":["DataNode","SingleFreeNode","GeneProduct"],"kaavioType":"SingleFreeNode","gpmlElementName":"DataNode"},"ea3a3":{"id":"ea3a3","textContent":"0005","wpType":"GeneProduct","x":18.771795096245356,"y":100.80986117035668,"fontSize":10,"height":20,"verticalAlign":"middle","width":80,"zIndex":32949,"textAlign":"center","color":"#000000","backgroundColor":"#ffffff","textDecoration":"none","fontFamily":"Arial","fontStyle":"normal","fontWeight":"normal","borderWidth":1,"drawAs":"rect","lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"type":["DataNode","SingleFreeNode","GeneProduct"],"kaavioType":"SingleFreeNode","gpmlElementName":"DataNode"},"f8809":{"id":"f8809","drawAs":"CurvedLine","borderWidth":1,"zIndex":32947,"points":[{"isAttachedTo":"ea3a3","attachmentDisplay":{"position":[0.5,0],"offset":[0,0]},"x":58.771795096245356,"y":100.80986117035668,"orientation":[0,-1]},{"x":119.28612545485854,"y":68.99683298432036},{"x":179.8004558134717,"y":117.31898063532702},{"x":212.1327356091221,"y":165.64112828633367},{"isAttachedTo":"ad716","attachmentDisplay":{"position":[0.5,1],"offset":[0,0]},"x":244.46501540477254,"y":130.62567176545974,"orientation":[0,-1]}],"color":"#000000","type":["Interaction","Edge"],"kaavioType":"Edge","gpmlElementName":"Interaction","markerStart":"TBar","markerEnd":"Arrow","isAttachedTo":["ea3a3","ad716"]}}}
